src/main/scala/com/exasol/cloudetl/parquet/converter/ParquetConverter.scala

```diff
@@ -364,8 +242,20 @@ sealed trait ArrayConverter {
     }
     elementConverter
   }
-  def start(): Unit = dataHolder.reset()
-  def end(): Unit = parentDataHolder.put(index, dataHolder.getValues())
+
+  def start(): Unit = {
+    dataHolder.reset()
+    if (elementConverter.isInstanceOf[RepeatedConverter]) {
```

Comment:
How about adding partentStartto the interface, all it every time and only implement it for `RepeatedConverter``.
I've the feeling that it's bad design to have a condition on the type here.

Reply:
Nice catch!
I have updated the converters, override only in repeated ones.